targets/frontend/src/lib/secu.ts

```diff
@@ -0,0 +1,24 @@
+import formidable from "formidable";
+
+export const isUploadFileSafe = (stream: formidable.Part): Promise<boolean> => {
+  return new Promise((resolve) => {
+    if (stream.mimetype !== "image/svg+xml") resolve(true);
+    stream.on("error", (err) => {
+      console.error("[storage]", err);
+      resolve(false);
+    });
+
+    let previousChunk = "";
+    let isSafe = true;
+    stream.on("data", (chunk) => {
+      previousChunk = chunk.toString();
+      const currentChunk = previousChunk + chunk.toString();
```

Comment:
```suggestion
      const currentChunk = previousChunk + chunk.toString();
      previousChunk = chunk.toString();
```

Si tu override le previous avant de le concaténer, tu fais chunk + chunk au final :)

Reply:
en effet 😅